src/anniversary/sf33rd/Source/Game/Flash_LP.c

```diff
@@ -0,0 +1,67 @@
+#include "common.h"
+#include "sf33rd/Source/Game/Flash_LP.h"
+#include "sf33rd/Source/Game/workuser.h"
+#include "sf33rd/Source/Game/WORK_SYS.h"
+#include "sf33rd/Source/Game/SysDir.h"
+#include "sf33rd/Source/Game/sc_data.h"
+
+extern const u8 Lamp_Flash_Data[2][2] = {{0x07, 0x6f}, {0x1e, 0x03}};
+
+void Flash_Lamp(void) {
+    unsigned char ix; // r16 (s0)
+    unsigned char ix2p; // r19 (s3)
+    unsigned char mark; // r18 (s2)
+    unsigned char color; // r17 (s1)
+
+    if (Mode_Type == 3 || Mode_Type == 4) {
+        return;
+    }
+
+    if (omop_cockpit == 0) {
+        return;
+    }
+
+    if (!Game_pause) {
+        switch (Lamp_No) {
+        case 0:
+            Lamp_No = 1;
+            Lamp_Index = 1;
+            Lamp_Timer = 1;
+        case 1:
+            if (--Lamp_Timer == 0) {
+                if (++Lamp_Index > 1) {
+                    Lamp_Index = 0;
+                }
+                Lamp_Color = Lamp_Flash_Data[Lamp_Index][0];
+                Lamp_Timer = Lamp_Flash_Data[Lamp_Index][1];
+            }
+        break;
```

Comment:
`break` is misaligned
```suggestion
            break;
```